resources/js/contest-judge-results/header.tsx

```diff
@@ -29,6 +29,7 @@ export default class Header extends React.PureComponent<Props> {
 
   render() {
     const totalScore = `${this.props.entry.results.votes}/${this.props.contest.max_total_score}`;
+    const totalScoreStd = this.props.entry.judge_votes.reduce((total, vote) => (vote.total_score_std ?? NaN) + total, 0);
```

Comment:
shouldn't the default when missing be 0 instead?

...or maybe just actually include and use the `this.props.entry.results.score_std` value